xml/System.Windows.Forms.Automation/IAutomationLiveRegion.xml

```diff
@@ -11,8 +11,15 @@
   </AssemblyInfo>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides support for UI automation live regions.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+This feature is available in applications that are recompiled to target .NET Framework 4.8 or that opt into this functionality by using compatibility switches.
```

Comment:
@merriemcgaw @Tanya-Solyanik Do we need to mention here what compatibility switch should be enabled to make it work?